irg_gazebo_plugins/materials/scripts/IRGCameraSim.frag

```diff
@@ -60,11 +60,12 @@ uniform vec3 offsets;
 // Standard deviation of the Gaussian distribution that we want to sample from.
 //uniform float stddev;
 
-// Read noise coefficient
-uniform float read_noise;
+// Read noise standard deviation (in integer pixel value where pixel values are
+// in the range {0, 2^adc_bits - 1} )
```

Comment:
I think curly braces are technically incorrect for range here, unless there are really only two valid values.  The square bracket (inclusive) and parenthesis (exclusive) notation seem appropriate here, i.e. either of:
```
[0, 2^adc_bits - 1]
[0, 2^adc_bits)
```

Reply:
Darn. I've seen this notation and have been writing ranges like this in documentation for years and nobody has ever said anything. I think in strict mathematics notation, it would also be called an "interval" instead of a "range." Not sure I want to switch to strict mathematical notation since this has never caused a problem before. Opinions?

Reply:
No strong opinion but the problem with the mathematical notation people may not observe that you are using two different brackets [) and even if they noticed they may think it is a merely a typo

Reply:
Changed 'range' to 'interval' and changed curly braces to square braces.